R/appender.R

```diff
@@ -184,3 +184,15 @@ appender.graylog <- function(server, port, debug = FALSE) {
     if (debug) print(ret)
   }
 }
+
+appender.kinesis_handler <- function(stream,region_name,partition_key)
+{
+
+  function(line) {
+      library(AWR.Kinesis)
```

Comment:
`requireNamespace` might be more appropriate here? And maybe adding to `Suggests` in the `DESCRIPTION` file.